examples/ensemble/plot_adaboost_multiclass.py

```diff
@@ -40,18 +40,18 @@
     n_samples=13000, n_features=10, n_classes=3, random_state=1
 )
 
-n_split = 3000
+n_split = 2000
```

Comment:
this is not reducing the number of splits, it's the size of the training set. You can instead make the size of the training set also smaller, and use train_test_split to get like 50% for training.

Reply:
How does this then have an impact on the execution time if it's just the relation train to test?

Reply:
I changed it back from 2000 to the original 3000. Even with only the ``n_estimators`` parameters changed, time reduction is from 16 sec to 7 sec